lisa/lisa-core/src/main/java/it/unive/lisa/LiSAConfiguration.java

```diff
@@ -86,6 +87,12 @@ public static enum GraphType {
 	 */
 	public static final int DEFAULT_WIDENING_THRESHOLD = 5;
 
+	/**
+	 * The default number of maximum time glb can be called on a node during the
+	 * descending phase of fixpoint algorithm.
+	 */
+	public static final int DEFAULT_DESCENDING_GLB_THRESHOLD = 5;
```

Comment:
I would make the documentation of this constant to be similar to the default widening threshold